File: Modul3/PRAK305-2310817110008-Muhammad_Raihan.c
#include <stdio.h>
#include <math.h>

void main () {
    int detik,menit,jam,hari,det_1hari;
    scanf("%d", &detik);
    det_1hari=24*pow(60,2);
    hari = detik/det_1hari;
    if (detik>=det_1hari) {
        printf("%d hari ", hari);
    }
    detik = detik - det_1hari*hari;
    jam = detik/pow(60,2);
    detik = detik - (pow(60,2)*jam);
    menit = detik/60;
    detik = detik - 60*menit;
    printf("%02d:%02d:%02d", jam,menit,detik);
}

File: Modul3/PRAK305-2310817110008-Muhammad_Raihan.py
det1 = int(input())
det_1hari = 24*pow(60,2)
hari = det1/det_1hari
detik = det1 - (det_1hari*int(hari))
jam = detik/pow(60,2)
detik = detik - (int(jam)*pow(60,2))
menit = detik/60
detik = detik - (60*int(menit))
if det1>=det_1hari:
    print("{:d} hari {:02d}:{:02d}:{:02d}".format(int(hari),int(jam),int(menit),int(detik)))
else:
    print("{:02d}:{:02d}:{:02d}".format(int(jam),int(menit),int(detik)))